tests/Integration/TestCase.php

```diff
@@ -135,7 +140,7 @@ public function create_test_taxonomy( $taxonomy_key, $term_name ) {
 	 * @return \ReflectionMethod
 	 * @throws \ReflectionException The method does not exist in the class.
 	 */
-	public static function getMethod( $method_name, $class_name = 'Parsely' ): \ReflectionMethod {
+	public static function getMethod( string $method_name, string $class_name = 'Parsely' ): \ReflectionMethod {
```

Comment:
The naming of this method is camelCase, yet all of the others are snake_case.

Could even potentially rename it to something slightly more descriptive.

Reply:
I'd also like to see the calls to this method updated to avoid the optional value on the second argument - that'll make the integration tests more understandable.

(And yes, I know I was the one who added the optional value recently!)